### PR TITLE
Spelling fix

### PR DESCRIPTION
**Pull request by sabahshariq**
## Summary

Fixed spelling and completed missing words in sentence.

### CODE REVIEW

**Review by douglaslMS — CHANGES_REQUESTED** [review]
@ThePiranha Thank you for catching these issues. In the 2nd of your 3 fixes, will you please change "users providing" to "users to provide" so it reads "...without requiring users to provide a user name or password." Thank you! Then I'll sign off to merge your fixes.

File: docs/framework/data/adonet/sql/authentication-in-sql-server.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ SQL Server supports two authentication modes, Windows authentication mode and mi
 > [!IMPORTANT]
 >  We recommend using Windows authentication wherever possible. Windows authentication uses a series of encrypted messages to authenticate users in SQL Server. When SQL Server logins are used, SQL Server login names and encrypted passwords are passed across the network, which makes them less secure.  
   
- With Windows authentication, users are already logged onto Windows and do not have to log on separately to SQL Server. The following `SqlConnection.ConnectionString` specifies Windows authentication without requiring the a user name or password.  
+ With Windows authentication, users are already logged onto Windows and do not have to log on separately to SQL Server. The following `SqlConnection.ConnectionString` specifies Windows authentication without requiring the users providing a user name or password.
```
Comment (mairaw) [inline docs/framework/data/adonet/sql/authentication-in-sql-server.md]:
```suggestion
 With Windows authentication, users are already logged onto Windows and do not have to log on separately to SQL Server. The following `SqlConnection.ConnectionString` specifies Windows authentication without requiring users to provide a user name or password.  
```

**Comment by mairaw** [comment]
Left @douglaslMS comment in form of suggestion so it's easier to apply the feedback.

**Review by douglaslMS — APPROVED** [review]

**Comment by ThePiranha** [comment]
image
for considering this PR.

**Pull request merged**